docs/DevelopmentPractices.md

```diff
@@ -49,6 +49,13 @@ A Program Evaluation Review Technique (PERT) Chart was created prior to the star
   <br><em>Figure 2</em>
 </p>
 
+A Gantt chart was created prior to the start of developing NotUS and was constantly updated based on progress and updates from the development team's weekly meetings. A Gantt chart is a visual view of tasks scheduled over time. The chart was used for planning project features and was a useful way of showing what work is scheduled and had to be completed.
```

Comment:
Maybe can add a line about how it ensured that people were "liable/responsible" (I cant exactly remember the word Im looking for). But basically, the idea that everyone knows that these are their "duties"